src/Common/test-runtime/project.json

```diff
@@ -1,384 +1,390 @@
 {
```

Comment:
Ah, I see.  It's more obvious with this view:
https://github.com/dotnet/corefx/pull/12805/files?w=1
Thanks